src/App.vue

```diff
@@ -42,14 +42,21 @@ import { useLoader } from "./services/use-loader";
 const contactmomentStore = useContactmomentStore();
 const route = useRoute();
 
-const { data: versienummer } = useLoader(() =>
-  fetchLoggedIn("/api/environment/versienummer")
+const { data: versieInfo } = useLoader(() =>
+  fetchLoggedIn("/api/environment/build-info")
     .then(throwIfNotOk)
     .then(parseJson)
-    .then(
-      ({ versienummer }: { versienummer?: string }) =>
-        versienummer?.split("+")?.[1] || "",
-    ),
+    .then(({ versionInfo }: { versionInfo?: string }) => {
+      const [version, commit] = versionInfo?.split("+") || [];
+      return {
+        version: version || "onbekend",
+        commit: commit?.replace(/^\./, "") || "onbekend",
```

Comment:
waarom is die replace nodig? zitten er punten in een commit hash en zijn die onhandig?